### PR TITLE
feat: insights out of beta and expose 2 widgets to pro

### PR DESCRIPTION
**Pull request by sjaanus**
1. Remove beta badge
2. Remove exposure from oss
3. Expose 2 widgets to pro and rest to enterprise

### CODE REVIEW

File: frontend/src/component/project/Project/Project.tsx [hunk 1]
```diff
@@ -93,7 +93,6 @@ export const Project = () => {
             title: 'Insights',
             path: `${basePath}/insights`,
             name: 'insights',
-            new: true,
```
Comment (sjaanus) [inline frontend/src/component/project/Project/Project.tsx:96]: Removes beta tag

File: frontend/src/component/menu/routes.ts [hunk 2]
```diff
@@ -153,7 +153,7 @@ export const routes: IRoute[] = [
         type: 'protected',
         menu: { mobile: true },
         notFlag: 'killInsightsUI',
-        enterprise: false,
+        enterprise: true,
```
Comment (sjaanus) [inline frontend/src/component/menu/routes.ts:156]: This is actually not enterprise, but pro and enterprise.

File: frontend/src/component/insights/InsightsCharts.tsx [hunk 3]
```diff
@@ -143,91 +145,115 @@ export const InsightsCharts: VFC<IChartsProps> = ({
...
-                        count={showAllProjects ? flags.total : summary.total}
-                        flagsPerUser={
-                            showAllProjects ? getFlagsPerUser(flags, users) : ''
-                        }
-                        isLoading={loading}
-                    />
-                </Widget>
                 <ConditionallyRender
-                    condition={showAllProjects}
+                    condition={isEnterprise()}
```
Comment (sjaanus) [inline frontend/src/component/insights/InsightsCharts.tsx:149]: Looks big change, but actually just adding extra conditional render.

File: frontend/src/component/insights/InsightsCharts.tsx [hunk 4]
```diff
@@ -22,6 +22,7 @@ import type {
 import type { GroupedDataByProject } from './hooks/useGroupedProjectTrends';
 import { allOption } from 'component/common/ProjectSelect/ProjectSelect';
 import { chartInfo } from './chart-info';
+import useUiConfig from '../../hooks/api/getters/useUiConfig/useUiConfig';
```
Comment (kwasniew) [inline frontend/src/component/insights/InsightsCharts.tsx:25]: dots :D

**Review by kwasniew — APPROVED** [review]

**Pull request merged**